public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -41,21 +49,21 @@ function createSidebar() {
     <div style="display: grid;" class="custom_field">
         <button onclick="rtdefault()" class="btn btn-large" style="border-radius: 0px;border: none;border-bottom: 1px solid black !important;">Default Settings</button>
     </div>
-    <p class="h_ar">Font Size:</p>
+    <!--<p class="h_ar">Font Size:</p>
     <div class="custom_field">   
        
         <input class="slider_ar" type="range" min=".4" max="1.8" value="1.1" step=".1" id="size"> 
 
 
     
-    </div>
+    </div>-->
     <p class="h_ar">Page Width:</p>
 <div class="custom_field">   
   <input class="slider_ar" type="range" min="0" max="450" value="0" step ="10" id="slider-page-width">
 </div>
    <p class="h_ar">Text Align:</p>
     <div class="custom_field"> 
-        <a id="toggler-text" href="#0" class="toggler">Left</a>
+        <a id="toggler-text" href="#0" class="toggler">Full</a>
```

Comment:
Thanks for swapping these labels! Hopefully people find it more intuitive but we'll see either way.